### DISCUSSION TITLE
Unable to Install

### DISCUSSION THREAD

**Discussion by Arunkumar2023** [nlmixr2/nlmixr2#129] [General]
Hi, I am a new user and trying to install nlmixr2 on my desktop. Encountering issue following is the error log, kindly help.

library(nlmixr2)
Loading required package: nlmixr2data
Registered S3 method overwritten by 'rxode2':
  method                 from       
  as.data.frame.rxEtTran rxode2parse
Error: package or namespace load failed for ‘nlmixr2’:
 .onLoad failed in loadNamespace() for 'rxode2', details:
  call: NULL
  error: rxode2 compiled with rxode2parse with a different solving structure
can try: install.packages('rxode2', type='source')
In addition: Warning message:
package ‘nlmixr2data’ was built under R version 4.2.3

**Comment by mattfidler** [comment]
We are waiting for CRAN to approve our packages, and they are out of sync, try the development version (which is close to the upcoming release version):

```r
install.packages(c("dparser", "nlmixr2data", "lotri", "rxode2ll",
                   "rxode2parse", "rxode2random", "rxode2et",
                   "rxode2", "nlmixr2est", "nlmixr2extra", "nlmixr2plot",
                   "nlmixr2"),
                 repos = c('https://nlmixr2.r-universe.dev',
                           'https://cloud.r-project.org'))
```

**Reply by Arunkumar2023** [reply]
Thanks for your quick reply, Matt. I will try this. Best regards, Arun
